### PR TITLE
Feat: Tracker page and about us button

### PR DESCRIPTION
**Pull request by Nourcova**
in this branch I created the Habit tracker page and  the "How it works" button in the Header now takes us to the about us section

### CODE REVIEW

File: src/Pages/TrackerPage/TrackerPage.js [hunk 1]
```diff
@@ -1,4 +1,57 @@
...
+              Completed
+            </p>{' '}
+          </div>
+          <div className=" bg-gray-300 rounded-full w-7 h-7 m-2 mx-28">
+            <p className="text-gray-800 text-s pl-8 mt-1 px-8">ongoing</p>
+          </div>
+        </div>
+
+        <div className="h-5/6 w-full items-center overflow-y-scroll p-4  mb-8 mt-1 rounded-lg mx-9  my-5  ">
+          {/*<HabitTrackerItem title="Brushing teeth"  duration= {[ { id: 1, status: 'checked', date: 'Sat Feb 20 2021' },
```
Comment (JanaKhanji) [inline src/Pages/TrackerPage/TrackerPage.js]: remove this cmnt

**Review by JanaKhanji — CHANGES_REQUESTED** [review]

File: src/Pages/TrackerPage/TrackerPage.js [hunk 2]
```diff
@@ -1,4 +1,57 @@
...
+  return (
+    <>
+      <div className="flex flex-col items-center mx-9 my-5 h-5/6 px-10">
+        <div className="flex justify-center  py-4 w-full rounded-xl shadow-md mx-8 h-20 bg-white px-7 mt-6">
+          <input
+            className="w-8/12 border-2 border-gray-400 p-4 mb-8 mt-1 rounded-lg"
+            type="text"
+            placeholder="Search"
+          />
+          <div className=" bg-primary rounded-full w-7 h-7 m-2">
```
Comment (JanaKhanji) [inline src/Pages/TrackerPage/TrackerPage.js]: make this a button and add onclick to add functionality later

File: src/Pages/TrackerPage/TrackerPage.js [hunk 3]
```diff
@@ -1,4 +1,57 @@
...
+            {' '}
+            <p className="text-gray-800 text-s pl-8 mt-1 px-8">
+              Completed
+            </p>{' '}
+          </div>
+          <div className=" bg-gray-300 rounded-full w-7 h-7 m-2 mx-28">
```
Comment (JanaKhanji) [inline src/Pages/TrackerPage/TrackerPage.js]: same here

**Review by JanaKhanji — CHANGES_REQUESTED** [review]

**Comment by osamaakb** [comment]
@Nourcova please fix the requested changes here

**Comment by osamaakb** [comment]
there is some padding here, remove it so the list is aligned on the left with the header
image

**Comment by osamaakb** [comment]
make this space between so the search bar goes to the most left and the buttons to the most right
image

**Comment by osamaakb** [comment]
Please add the functionality to this page too

**Comment by osamaakb** [comment]
please fix the conflicts and the netlify errors

File: src/Pages/HomePage/Header/Header.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ import { useTranslation } from 'react-i18next';
 import i18next from 'i18next';
 import { Link } from 'react-router-dom';
 
-function Header() {
+function Header({ func }) {
```
Comment (osamaakb) [inline src/Pages/HomePage/Header/Header.js:14]: what's this `func` here

**Review by osamaakb — APPROVED** [review]

**Pull request merged**